hosts/hosts.go

```diff
@@ -29,6 +29,7 @@ const (
 	ToCleanCNIConf       = "/etc/cni"
 	ToCleanCNIBin        = "/opt/cni"
 	ToCleanCalicoRun     = "/var/run/calico"
+	ToCleanTempCertPath  = "/etc/kubernetes/.tmp/"
```

Comment:
Can you add this constant to the clean hosts functions

Reply:
done